libvmi/cache.c

```diff
@@ -80,7 +81,7 @@ void key_128_init(vmi_instance_t vmi, key_128_t key, uint64_t low, uint64_t high
 
 key_128_t key_128_build (vmi_instance_t vmi, uint64_t low, uint64_t high)
 {
-    key_128_t key = (key_128_t) g_malloc0(sizeof(struct key_128));
+    key_128_t key = (key_128_t) g_malloc(sizeof(struct key_128));
```

Comment:
Why the switch from `g_malloc0`?

Reply:
Speed - we know that we're writing to each struct member. If you don't like this I can change it back.

Reply:
Fair enough